### ISSUE TITLE
Releases missing from maven central?

### ISSUE THREAD

**Issue by skennedy** [scala-ts/scala-ts#6]
Followed your instructions to add the plugin to sbt but it is failing to resolve. Also noticed all the links to pom / jar on this page return 404s:

http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.github.miloszpp%22%20AND%20a%3A%22scala-ts%22

**Comment by miloszpp** [comment]
Thank you for reporting this. I will investigate and let you know when it's fixed.

**Comment by miloszpp** [comment]
I was told by Joel from OSSRH that it's browsable here:
http://repo1.maven.org/maven2/com/github/miloszpp/scala-ts_2.10_0.13/0.3.0/

As a temporary workaround, you can try downloading and using in the standalone mode.

@nicolasdalsass could you please confirm that you are able to resolve the plugin?

**Comment by nicolasdalsass** [comment]
Yes, I could resolve a few minutes ago.

**Comment by Troxid** [comment]
Hello, when i add `addSbtPlugin("com.github.miloszpp" % "scala-ts" % "0.4.0")` to the `plugin.sbt` i get the error:
```
UNRESOLVED DEPENDENCIES
com.github.miloszpp#scala-ts;0.4.0: not found
Note: Some unresolved dependencies have extra attributes.  Check that these dependencies exist with the requested attributes. com.github.miloszpp:scala-ts:0.4.0 (scalaVersion=2.12, sbtVersion=1.0)
```
The same error occurs if i add `libraryDependencies += "com.github.miloszpp" % "scala-ts" % "0.4.0"` to the `build.sbt`
Does the plugin require special resolver?

**Comment by Troxid** [comment]
Also, package in the maven central repository does't have jar/pom files.
`http://search.maven.org/remotecontent?filepath=com/github/miloszpp/scala-ts/0.4.0/scala-ts-0.4.0.jar`
return 404

**Comment by rators** [comment]
any update on this?

**Comment by rators** [comment]
project is not usable at the moment

**Comment by lbialy** [comment]
https://repo1.maven.org/maven2/com/github/miloszpp/ < Miłosz, you basically published it just for 2.10.
